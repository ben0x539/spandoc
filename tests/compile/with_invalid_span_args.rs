#[spandoc::spandoc]
fn main() {
    /// SPANDOC: Show me the money {?local}
    let _ = 4;
}
