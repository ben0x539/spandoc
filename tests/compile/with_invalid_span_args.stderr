error[E0425]: cannot find value `local` in this scope
 --> $DIR/with_invalid_span_args.rs:3:5
  |
3 |     /// SPANDOC: Show me the money {?local}
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
